app/models/katello/sync_plan.rb

```diff
@@ -55,6 +55,10 @@ def plan_date_time
       self.sync_date.strftime('%Y/%m/%d %H:%M:%S %Z')
     end
 
+    def sync_date_with_offset
```

Comment:
Should this be `utc_offset` to help clarify the intent is to display this with the UTC based offset?